Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteLayerTreeDrawingAreaProxyIOS.h

```diff
@@ -29,7 +29,7 @@
 
 #if PLATFORM(IOS_FAMILY)
 
-OBJC_CLASS WKOneShotDisplayLinkHandler;
+OBJC_CLASS WKDisplayLinkHandler;
```

Comment:
Why not WKDisplayRefreshHandler to be consistent?

Reply:
Could be, but it wraps a `CADisplayLink`.

Reply:
Hm... I still think WKDisplayRefreshHandler is a more descriptive name, not that I would insist on it.